### PR TITLE
Enhance readme

### PR DESCRIPTION
**Pull request by guidotripaldi**
Add step-by-step instructions to how to run the
Openmaize-Phoenix-Example.

Previous version describe how to get started with the Openmaize library,
but lacks hints to  start with the Phoenix demo Application.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,90 @@ The latest version of Openmaize depends on Elixir 1.2.
...
 application.
 
-## Getting started with Openmaize
+
+## Getting started with the Openmaize-Phoenix-Example `Welcome` Application
+
+These are the step-by-step instructions to how to run this example for the first time:   
+
+ 1. Adapt the configuration files in `config/` and 
+    the file `lib/welcome/mailer.ex` to reflect your local enviroiment.
```
Comment (jaimeiniesta) [inline README.md]: Typo: "enviroiment" => "environment".

Comment (guidotripaldi) [inline README.md]: Thanks

**Comment by riverrun** [comment]
Sorry for the delay in getting back to you.
Thanks for the contribution - the README looks a lot better now.

**Comment by guidotripaldi** [comment]
sorry for the messy commits, I've pushed before squashing them together..

**Pull request merged**